README.md

```diff
@@ -269,11 +269,12 @@ When running the processes, the following commands can be used to test functiona
 | `USER_ADDRESS` | An account - the current owner of coins/tokens. |
 | `USER_ADDRESS_PRIVATE_KEY` | A private key belonging to the account. |
 | `HOME_BRIDGE_ADDRESS` | Address of the bridge in the Home network to send transactions. |
-| `HOME_MIN_AMOUNT_PER_TX` | Value (in _eth_ or tokens) to be sent in one transaction for the Home network. |
+| `HOME_MIN_AMOUNT_PER_TX` | Value (in _eth_ or tokens) to be sent in one transaction for the Home network. This should match the value specified in the `poa-bridge-contracts/deploy/.env` file. The default value in that file is 500000000000000000, which is equivilant to 0.5   |
 | `FOREIGN_BRIDGE_ADDRESS` | Address of the bridge in the Foreign network to send transactions. |
-| `FOREIGN_MIN_AMOUNT_PER_TX` | Value (in _eth_ or tokens) to be sent in one transaction for the Foreign network. |
+| `FOREIGN_MIN_AMOUNT_PER_TX` | Value (in _eth_ or tokens) to be sent in one transaction for the Foreign network. This should match the value specified in the `poa-bridge-contracts/deploy.env` file. The default value in that file is 500000000000000000, which is equivilant to 0.5 |
```

Comment:
```suggestion
| `FOREIGN_MIN_AMOUNT_PER_TX` | Value (in _eth_ or tokens) to be sent in one transaction for the Foreign network. This should be greater or equal to the value specified in the `poa-bridge-contracts/deploy.env` file. The default value in that file is 500000000000000000, which is equivalent to 0.5 |
```

Reply:
Also, it says `deploy.env` when it should be `deploy/.env`.